src/test/java/guitests/BoardAutoCreatorTest.java

```diff
@@ -104,15 +104,23 @@ public void sampleBoardAutoCreationTest() {
 
         waitUntilNodeAppears(SAMPLE_BOARD_DIALOG);
         click("OK");
+        verifyBoard(panelControl, BoardAutoCreator.getSamplePanelDetails());
+    }
 
-        assertEquals(panelControl.getPanelCount(), SAMPLE_PANEL_NAMES.size());
-        assertEquals(panelControl.getCurrentlySelectedPanel(), Optional.of(0));
-        assertEquals(panelControl.getNumberOfSavedBoards(), 1);
-
-        List<PanelInfo> panelInfos = panelControl.getCurrentPanelInfos();
-        for (int i = 0; i < SAMPLE_PANEL_NAMES.size(); i++){
-            assertEquals(panelInfos.get(i).getPanelFilter(), SAMPLE_PANEL_FILTERS.get(i));
-            assertEquals(panelInfos.get(i).getPanelName(), SAMPLE_PANEL_NAMES.get(i));
+    /**
+     * Tests panelControl properties associated with panels.
+     * panelDetails obtained from Boards are also tested as a subset of the panels in PanelControl.
```

Comment:
Why a subset? I said 'subset' earlier because that was what the method was doing. Is this method checking for subset or exact match?